packages/web-components/fast-foundation/src/disclosure/disclosure.ts

```diff
@@ -0,0 +1,117 @@
+import { attr, FASTElement } from "@microsoft/fast-element";
+/**
+ * A Disclosure Custom HTML Element.
+ * Based largely on the {@link https://w3c.github.io/aria-practices/#disclosure | disclosure element }.
+ *
+ * @public
+ */
+export class Disclosure extends FASTElement {
+    /**
+     * See {@link https://www.w3.org/WAI/PF/aria/roles#button} for more information
+     * @public
+     * @remarks
+     * HTML Attribute: aria-expanded
+     */
+    @attr({ attribute: "aria-expanded" })
+    public ariaExpanded: true | false;
+
+    /**
+     * Identifies the element (or elements) whose contents or presence are controlled by the current element.
+     *
+     * {@link https://www.w3.org/TR/wai-aria-1.1/#aria-controls}
+     * @public
+     * @remarks
+     * HTML Attribute: aria-controls
+     */
+    @attr({ attribute: "aria-controls" })
+    public ariaControls: string;
```

Comment:
Does this need to be publicly available or can it not just be set to the value of an `id` provided by the user? Alternatively, the Shadow DOM id should be scoped so you could technically bake this in...

Reply:
Remove it.